sdk/identity/Azure.Identity.BrokeredAuthentication/tests/Azure.Identity.BrokeredAuthentication.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks);netcoreapp3.1;net461</TargetFrameworks>
```

Comment:
are the extra targets necessary? I thought we'd get them all from RequiredTargetFrameworks

Reply:
I don't think they are. By default, I believe only netstandard2.0 is targeted. We need to build for these other targets as well to since we have #if directives which are conditionally compiled in for these frameworks.

Reply:
But if you remove them, we don't still build those framework versions?

Reply:
No if we don't explicitly include them the package will only have a single assembly for netstandard2.0.